pages/resume.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react'
+import Resume from '../components/resume/resume'
+import jbatorski from '../data/resume/jbatorski.json'
+
+interface ResumePageProps {
+    data: any
+}
+
+const ResumePage: FC<ResumePageProps> = ({ data }) => {
+    return <Resume data={data} />
+}
+
+export async function getStaticProps() {
+
+    return {
+        props: {
+            data: jbatorski,
```

Comment:
Later on can prob move this out into a function which would simulate getting the resume data from other sources, or even allow visitors of this site to upload theirs to get it rendered like mine is.

Reply:
Added this as a tracker item to https://github.com/jarekb84/me/issues/1